### ISSUE TITLE
We should have a link or share button to invite people to the community, rather than admin having to copy and past emails. 

### ISSUE THREAD

**Issue by MaxTattenbach** [opencurrents/opencurrents#219]
Note there should be an easy way for people to share a link and share it on their FB page or others to join openCurrents.   This should be an option for both Volunteers and Admins. 

![issue 4 not showing profile name](https://user-images.githubusercontent.com/26234440/28691777-b7e631be-72e3-11e7-80c3-7fdd491a2085.PNG)

**Comment by dannypernik** [comment]
@MaxTattenbach agreed. do you think "Invite volunteers" is the page for this? Currently only admins have access to invite-volunteers. What about on the profile? @nickolashe

**Comment by nickolashe** [comment]
what's the status of this one?

**Comment by dannypernik** [comment]
Added to to-do list

**Comment by nickolashe** [comment]
@dannypernik thanks, can you provide an estimate (in hours) for this task?

**Comment by dannypernik** [comment]
@nickolashe  I've tried setting up Facebook sharing before and it was more complicated than I expected. Maybe 2-3 hours.

**Comment by nickolashe** [comment]
@dannypernik lets use the Estimate PR tag to convey this
